src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person:
+  { name, age, sex, isMarried, partnerName } }) => (
+
+    <section className="Person">
+      {name && (
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      )}
```

Comment:
You also need to use `defaultProps` and `propTypes` in this task.
And `name` I think should be required. This way you wouldn't need this check.